role-service/src/main/resources/bootstrap-local.yml

```diff
@@ -6,3 +6,11 @@ spring:
         enabled: false
     discovery:
       enabled: false
+  liquibase:
```

Comment:
Can all these be moved to app local file?

Reply:
Done